WordPress/Classes/Models/Blog+QuickStart.swift

```diff
@@ -25,6 +25,19 @@ extension Blog {
         ContextManager.sharedInstance().saveContextAndWait(context)
     }
 
+    public func removeAllTours(onError: (NSError) -> ()) {
+        guard let quickStartTours = quickStartTours else {
```

Comment:
Do you need the `onError: (NSError) -> ()` closure here?

Reply:
Nah, I think it can be removed 👍